doc/README.open_ce_build.md

```diff
@@ -410,3 +416,37 @@ The Dockerfile for this runtime image is located in [`images/opence-runtime/Dock
 This file can also be built and run manually and supports GPUs if the system is set up with
 `nvidia-container-runtime`. When building the Dockerfile directly, it does require a few arguents,
 check the Dockerfile for details.
+
+### Building packages with Power10 MMA optimization
+
+#### System Requirement
+System: RHEL 8.5 or above
+OS: Linux
+Power Architecture: Power9/Power10
+GCC Compiler: GCC10 and GCC11
+
+One can build Power10 enabled packages on Power9 or Power10 with above system requirements. To install GCC 10 or 11, following command can be used -
+```shell
+    yum install -y gcc-toolset-10 gcc-toolset-11
+```
+
+Set GCC_10_HOME and GCC_11_HOME environment variables to proceed with the builds on baremetal, if GCC10 or GCC11 is installed at a non-default location.
+
+For example:
+```shell
+    export GCC_10_HOME=/opt/rh/gcc-toolset-10/root/usr
+```
+
+Currently GCC 11 is used only to build [`openblas-feedstock`](https://github.com/open-ce/openblas-feedstock). All other Open-CE recipes can be built using GCC 10.
+
+GCC 10/11 setup is automated if the builds are done in a podman container using `--container_build` option. Please see [`Dockerfile`](https://github.com/open-ce/open-ce-builder/blob/main/open_ce/images/builder/Dockerfile-p10) used for containerized build of these packages.
+
+#### Build packages
+Power10 MMA Optimization is applicable for cpu only builds. One has to use `--ppc_arch=p10` flag in the `open-ce build env` or `open-ce build feedstock` command to build P10 enabled packages. Another important argument which is must to build these packages is `--conda_build_config=open-ce/envs/conda_build_config.yaml,open-ce/envs/conda_build_config_p10.yaml`.[`conda_build_config_p10.yaml`](https://github.com/open-ce/open-ce/blob/main/envs/conda_build_config_p10.yaml) contains Power10 specific settings.
+
+For example:
+```shell
+    open-ce build env --build_type=cpu --ppc_arch=p10 --conda_build_config=open-ce/envs/conda_build_config.yaml,open-ce/envs/conda_build_config_p10.yaml tensorflow-env.yaml
+```
+
+Packages built with above commands runs on Power10 with MMA optimization and on Power9 without MMA optimization. GCC10 is also needed at runtime for some of these packages to work, may it be P9 or P10.
```

Comment:
> Packages built with above commands runs

Packages built with above commands run~s~

I suggest to put _GCC10 is also needed at runtime for some of these packages to work, may it be P9 or P10._ into its own section, like so

### Using packages with Power10 MMA optimization

When using packages that were built with `ppc_arch=p10`, note that:
- These packages will work on Power9 or Power10, but not on Power8
- When using `PyTorch` or `sentencepiece`, gcc10 needs to be present on the system. These two packages require some gcc10 libraries to be present at runtime. This applies to both Power9 and Power10 systems.